packages/toolpad-components/src/Checkbox.tsx

```diff
@@ -63,6 +63,19 @@ function Checkbox({ ...rest }: Props) {
     [onFormInputChange],
   );
 
+  const props = React.useMemo(
```

Comment:
The `useMemo` is not necessary here, we're spreading these props to a component anyway.